### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by dcadenas**
A PR template. Covers https://github.com/planetary-social/nos/issues/1072

### CODE REVIEW

File: .github/pull_request_template.md
Comment (joshuatbrown) [inline .github/pull_request_template.md:2]:
```suggestion
A description of the changes proposed in the pull request. Explain what it does to give the reviewer some context for what they're reviewing.
```

File: .github/pull_request_template.md
Comment (joshuatbrown) [inline .github/pull_request_template.md:11]: I'd prefer to have this at the very top.

File: .github/pull_request_template.md
Comment (joshuatbrown) [inline .github/pull_request_template.md:4]:
```suggestion
## How to test
1. Navigate to...
2. Tap...
```

Just want to make sure it's clear what we expect here.

File: .github/pull_request_template.md
Comment (joshuatbrown) [inline .github/pull_request_template.md:7]:
```suggestion
## Screenshots/Video
```

File: .github/pull_request_template.md
Comment (joshuatbrown) [inline .github/pull_request_template.md:8]:
```suggestion
Post screenshots or video showing your changes, ideally showing how the app worked before and after these changes. Delete this section if this PR contains no visual changes.
```

File: .github/pull_request_template.md
Comment (joshuatbrown) [inline .github/pull_request_template.md:10]:
```suggestion
## Issues covered
```

**Review by joshuatbrown — CHANGES_REQUESTED** [review]
This is great! Thanks for getting this started.

File: .github/pull_request_template.md
Comment (joshuatbrown) [inline .github/pull_request_template.md:10]:
```suggestion
```

**Review by joshuatbrown — CHANGES_REQUESTED** [review]
One more suggestion, then I'm happy with this. I hope this leaves the "How to test" section clear enough.

**Comment by joshuatbrown** [comment]
@dcadenas you can also add the Skip Changelog label to this PR to get that check to pass; this one doesn't require a changelog update.

File: .github/pull_request_template.md
Comment (mplorentz) [inline .github/pull_request_template.md:13]: I don't find this section useful as a reviewer, because I'm going to build and run the app to see for myself. It's also time consuming to create. Are others finding value in this? @martindsq @dcadenas @joshuatbrown

File: .github/pull_request_template.md
Comment (mplorentz) [reply .github/pull_request_template.md:10]: If we are going to write down test steps maybe it makes sense to copy them to the ticket somehow so @setch-l can also use them for UAT?

Comment (mplorentz) [reply .github/pull_request_template.md:10]: idk I don't necessarily want to create a heavier process or have to manually copy text and then have it duplicated in two places 🤔 maybe I regret making this comment.

File: .github/pull_request_template.md
Comment (joshuatbrown) [reply .github/pull_request_template.md:13]: I do use it as a reviewer. I always review the code first, before building and running. When there are screenshots or video, I have a better idea of what I'm reviewing.

As a PR author, I don't find it time consuming to create. I generally test manually a bunch of times; during one of those, I hit the Save Screen or Record Screen option in the Simulator and upload it to the PR.

**Review by mplorentz — APPROVED** [review]
This looks good to me, but let's wait for 3 approvals before merging this since it is kind of a team process change.

**Review by joshuatbrown — APPROVED** [review]
Looks great to me!

**Comment by dcadenas** [comment]
@martindsq good to go?

File: .github/pull_request_template.md
Comment (martindsq) [reply .github/pull_request_template.md:12]: I think this is a good idea as well

**Review by martindsq — APPROVED** [review]

**Pull request merged**
